src/ForkCMS/Bundle/InstallerBundle/Console/InstallModuleCommand.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+namespace ForkCMS\Bundle\InstallerBundle\Console;
+
+use Backend\Core\Engine\Model as BackendModel;
+use Backend\Modules\Extensions\Engine\Model;
+use Backend\Modules\Extensions\Engine\Model as BackendExtensionsModel;
+use Doctrine\DBAL\Connection;
+use Doctrine\ORM\EntityManager;
+use Exception;
+use ForkCMS\App\BaseModel;
+use PDO;
+use RuntimeException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * This command allows you to install a module via the CLI
+ */
+class InstallModuleCommand extends Command
+{
+    /** @var SymfonyStyle */
+    private $formatter;
+
+    /** @var Connection */
+    private $dbConnection;
+
+    /** @var KernelInterface */
+    private $kernel;
+
+    public function __construct(EntityManager $em, KernelInterface $kernel)
+    {
+        parent::__construct();
+        $this->dbConnection = $em->getConnection();
+        $this->kernel = $kernel;
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('forkcms:install:module')
+            ->setDescription('Command to install a module in Fork CMS')
+            ->addArgument('module', InputArgument::OPTIONAL, 'Name of the module to install');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $this->formatter = new SymfonyStyle($input, $output);
+        $module = $this->getModuleToInstall($input, $output);
+
+        if (BackendExtensionsModel::existsModule($module)) {
+            // Make sure this module can be installed
+            $output->writeln("<comment>Validating if module can be installed...</comment>");
+            $this->validateIfModuleCanBeInstalled($module);
+
+            // Reboot the kernel to trigger a kernel initialize which registers the Dependency Injection Extension of the
+            // module we would like to install. Also, make sure to replace the static cached container in our BaseModel!
+            $_SERVER['INSTALLING_MODULE'] = $module;
```

Comment:
It took me a while to understand that in the `Kernel.php` we explicitely listen to the exact route which installs a module, and then we register the DI Extension file of the module. 

I need a way to re-initialize the kernel and clearing the cache did not seem to work, so I reboot the Kernel. 

Also needed a way to pass the module name back to the `Kernel.php` file to pick up the module name and initialize the DI Extension. Feel free to suggest better options than putting it in `$_SERVER` ? The whole register-module-extension-on-install feels a bit hacky unfortunately